### ISSUE TITLE
Runaway

### ISSUE THREAD

**Issue by KnuckleUpFPV** [dbasch/betaflight#3]
I was testing the GPS return to home over the last two days. And today is the day I took it the furthest out. Made it to 1500ft away and activated the return to home function. The quad immediately went into runaway mode. It should have made it well passed the runaway disable time limit. I had already surpassed the 25% throttle point and had been in the air for around a minute. Can I just disable the runaway function so this doesn't happen again? Thanks in advance.

Oh and I saw another video on YouTube where another person was having the same issue.

**Comment by s0up** [comment]
Do you have any DVR of this issue?  That would be very helpful 👍

**Comment by KnuckleUpFPV** [comment]
This is not my video. I'm not home to upload mine now. Go to1:50 of this video. I've tried it around 10 to 15 times from different areas. In all cases I was going slow. Until this last time. I was in forward flight moving along just like the person in this video. I flicked the switch and it immediately went into runaway and tumbled down. I didn't think to disarm and re arm to try and save it. 


https://youtu.be/O34G0aTe0DM

**Comment by s0up** [comment]
Need to reduce max throttle by a lot.  Previous builds had a very high
setting for that.  You'd want to increase the ground speed / tilt settings
and reduce throttle max.  Type get rescue to see all of the settings.

Also it will climb to whatever the highest altitude there was during your
pre rth flight. 👍

**Comment by KnuckleUpFPV** [comment]
I want going fast just slow cruising forward. Not my flight in the video. As soon as I flicked the return home switch the quad showed runaway on the odd and tumbled out of the air. It didn't even attempt to make the usual yaw turn to face home. All the settings are default from the Diegofpv.com binary. I disabled runaway mode. Then put it in the air to make sure return to home still worked. Which it did. I will have to test some more to make sure it doesn't kick back into runaway mode. If it's a speed thing I hope something is posted somewhere about turning down max speed so your quad doesn't auto disarm in runaway mode. I got lucky and didn't break anything. Once this goes more widespread I can see a lot more people breaking stuff, because the quad went into runaway mode and disarmed. I didn't even know it was a possibility until it happened. The way I read runaway mode works it should have been disabled at that point in my flight. Doesn't make sense to me why it was still active. Unless the return to home is somehow restarting the runaway function.

**Comment by s0up** [comment]
Can you get a DVR of it happening possibly?  We have nothing in place for
anything known as runaway mode.  Not sure what is up with that.

**Comment by KnuckleUpFPV** [comment]
Yeah I'm home and have it uploaded now. Placing link here. As soon as I flip the switch you will see runaway show up on the osd. Just like the video link i provided you with earlier. It works awesome when it works. When it doesn't you fall from the sky. I appreciate all that you all are doing for the community, and want to thank you for looking into this. 

https://youtu.be/w2GIfgno4G8

**Comment by s0up** [comment]
Oh!! That is runaway detection (anti Taz mode) which you can turn off.
That isn't related to GPS rescue but is actually triggered by the abrupt
yaw it does.  If you're on an older rescue nightly you can lower nav_p to
make it not yaw so hard, and on newer nightlies there is a setting called
gps_rescue_yaw_P.

Basically your craft is yawing so fast betaflight triggers the anti taz
mode.  Hopefully that clears it up for you.  I'm really glad you're
enjoying it -- super excited to really get it dialed in.

**Comment by KnuckleUpFPV** [comment]
Well that solves it. I already turned that crap off. I was mistaken in thinking it was off by default. When it popped up on the GPS return trigger I wasn't sure if it was vital to the GPS rescue. Now I feel much better. Going to hook up all 14 big quads and turn it off on all of them. Thanks for your time. I will continue to test this out and let you know of any other issues I encounter. Feel free to close this. Thanks again.
